src/query/anonymization.c

```diff
@@ -310,11 +323,9 @@ static Node *rewrite_to_avg_noise_aggregator(Aggref *aggref, List *aid_refs)
   aggref->aggtype = FLOAT8OID;
   aggref->aggstar = false;
   aggref->aggdistinct = false;
-
-  Aggref *count_aggref = make_anon_count_value_aggref(aggref);
-
   append_aid_args(aggref, aid_refs);
-  append_aid_args(count_aggref, aid_refs);
+
+  Expr *count_aggref = make_safe_anon_count_value(aggref);
```

Comment:
The word "safe" can have multiple meanings. Maybe the callers don't need to care about the subtle behavior at count=0?

Reply:
yeah, I was wondering about the name here. I'd love to call it `make_anon_count_value` but that's not what that is. I also would like to keep the addition of `nullif` together with building of the `count`, so that it's harder to miss it.

It is a "dedicated" `count` instance, so I need a name that conveys the idea :thinking: `make_avg_denominator_count_expr` ?

Reply:
Yes, or `make_count_denominator` for short is ok.

Reply:
`make_count_denominator` is nice, thx!